File: My algorithms collection/Euclidean Algorithm/Euclidean_algo.js
function gcd(a, b) { //the greatest common divisor
    while (b !== 0) {
        let temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

let a = 56, b = 98;
console.log(`GCD of ${a} and ${b} is ${gcd(a, b)}`);

File: My algorithms collection/Euclidean Algorithm/Euclidean_algo.c
#include <stdio.h>

int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    int a = 56, b = 98;
    printf("GCD of %d and %d is %d\n", a, b, gcd(a, b));
    return 0;
}
